recipes-core/images/rpi-basic-image.bb

```diff
@@ -1,5 +1,10 @@
-# Base this image on rpi-hwup-image
-include rpi-hwup-image.bb
+# Base this image on core-image-minimal
+include recipes-core/images/core-image-minimal.bb
```

Comment:
I think it should use core-image-base, which is more usable then core-image-minimal

Reply:
I tried to go with minimal change from the current behaviour and rpi-hwup-image is based on core-image-minimal. They're both deprecated now though so I don't think it's worth changing.

Reply:
OK thats fine by me